src/interpreter/vm/VM.cpp

```diff
@@ -28,15 +28,38 @@ int VM::run() {
         unsigned char opcode = nextBytecode();
         DURIAN_DEBUG_LOG("%x\n", opcode);
         DurianObject a, b;
+        int32_t i;
```

Comment:
Not immediately obvious what `i` is for... might be better to rename it since it has to be defined so far from use.